### PR TITLE
CP-38530: Remove PuTTY from build scripts and wix installer

### PR DESCRIPTION
**Pull request by danilo-delbusso**
To me merged with https://github.com/xenserver/dotnet-packages/pull/93 as part of CP-38530

Currently awaiting review and merge of https://github.com/xenserver/xenadmin/pull/2909.

The only relevant commit is ~59d7a5e1e549ca23a7598047c85198f71942582d~ 3020e93 (and onwards if changes are made).

### CODE REVIEW

**Comment by kc284** [comment]
Can I confirm that this depends on https://github.com/xenserver/xenadmin/pull/2909 being merged first?

**Comment by danilo-delbusso** [comment]
> Can I confirm that this depends on #2909 being merged first?

Yes that's correct @kc284

File: XenAdmin/Properties/Settings.settings [hunk 1]
```diff
@@ -197,5 +197,14 @@
     <Setting Name="RemindChangePassword" Roaming="true" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">True</Value>
     </Setting>
+    <Setting Name="CustomSshConsole" Type="XenAdmin.Core.SSHClient" Scope="User">
```
Comment (AaronRobson) [inline XenAdmin/Properties/Settings.settings:200]: Should these have `Roaming="true"`?

Comment (danilo-delbusso) [reply XenAdmin/Properties/Settings.settings]: apologies for the possible confusion. The only relevant commit is 3020e93 (and onwards if changes are made).

**Review by AaronRobson — DISMISSED** [review]

**Review by CitrixChris — APPROVED** [review]

**Review by kc284 — CHANGES_REQUESTED** [review]
We also need to remove the licence from the LegalNoticesDialog.

**Comment by danilo-delbusso** [comment]
> Looks ok to me now, but we'll need to get a build from [xenserver/dotnet-packages#93](https://github.com/xenserver/dotnet-packages/pull/93) before merging.

Build is ready. Project builds as expected after downloading the packages from the new location.

**Review by kc284 — APPROVED** [review]

File: XenAdmin/Dialogs/LegalNoticesDialog.resx [hunk 2]
```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+
```
Comment (AaronRobson) [inline XenAdmin/Dialogs/LegalNoticesDialog.resx:29]: I presume all this extra whitespace is Visual Studio messing about?

**Review by AaronRobson — APPROVED** [review]

File: XenAdmin/Dialogs/LegalNoticesDialog.resx (hunk 2 above)
Comment (danilo-delbusso) [reply XenAdmin/Dialogs/LegalNoticesDialog.resx:29]: yes this was automatic. I didn't even touch the code. I simply used VS' form for changing strings. Tried this a few times with the same result.

**Pull request merged**